orange3/si.yaml

```diff
@@ -9920,71 +9920,71 @@ widgets/unsupervised/owdistancematrix.py:
+    Mahalanobis: "Mahalanobisova"
+    Bhattacharyya: "Bhattacharyyeva"
     class `DistanceRunner`:
         def `run`:
-            Calculating...: null
-            axis: null
-            impute: null
-            callback: null
-            normalize: null
+            Calculating...: "Računam ..."
+            axis: false
+            impute: false
+            callback: false
+            normalize: false
     class `OWDistances`:
-        Distances: null
-        Compute a matrix of pairwise distances.: null
-        icons/Distance.svg: null
+        Distances: "Razdalje"
+        Compute a matrix of pairwise distances.: "Izračunaj parno matriko razdalj."
+        icons/Distance.svg: false
         class `Inputs`:
-            Data: null
+            Data: false
         class `Outputs`:
-            Distances: null
+            Distances: false
         class `Error`:
-            No numeric features: null
-            No binary features: null
-            '{} requires dense data.': null
-            Not enough memory: null
-            Problem in calculation:\n{}: null
-            Mahalanobis handles up to 1000 {}.: null
+            No numeric features: "Ni številskih spremenljivk"
+            No binary features: "Ni binarnih spremenljivk"
+            '{} requires dense data.': "{} zahteva goste podatke."
+            Not enough memory: "Ni dovolj spomina"
+            Problem in calculation:\n{}: 'Težava v izračunu:\n{}'
+            Mahalanobis handles up to 1000 {}.: 'Mahalanobisova razdalja zmore do 1000 {}.'
         class `Warning`:
-            Ignoring categorical features: null
-            Ignoring non-binary features: null
-            Missing values were imputed: null
-        def `__init__`:
-            axis: null
-            Rows: null
-            Columns: null
-            Distances between: null
-            Distance Metric: null
-            metric_idx: null
-            normalized_dist: null
-            Normalized: null
-            All dimensions are (implicitly) scaled to a common: null
-            scale to normalize the influence across the domain.: null
-            autocommit: null
+            Ignoring categorical features: "Kategorične spremenljivke niso upoštevane"
+            Ignoring non-binary features: "Ne-binarne spremenljivke niso upoštevane"
+            Missing values were imputed: "Manjkajoče vrednosti so bile pripisane"
+        def `__init__`:
+            axis: false
+            Rows: "Vrsticami"
+            Columns: "Stolpci"
+            Distances between: "Razdalje med"
+            Distance Metric: "Mera razdalj"
+            metric_idx: false
+            normalized_dist: false
+            Normalized: "Normalizirano"
+            All dimensions are (implicitly) scaled to a common: "Vse dimenzije so (implicitno) prilagojene skupni "
+            scale to normalize the influence across the domain.: "skali, s čimer se normalizira vpliv spremenljivk v domeni."
```

Comment:
Boljše bi se bilo (tudi v angleščini, ampak pustimo to tu) izogniti besedi "domena". Preprosto izpusti "v domeni". 

Jaz bi napisal ... `"Vse spremenljivke so preslikane na enak razpon, s čimer se izenači njihov vpliv na izračunano razdaljo."`